ViewModel/Saving/PersistenceManagers/AlternativeComparisonReportPersistenceManager.cs

```diff
@@ -0,0 +1,88 @@
+﻿using FdaLogging;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using ViewModel.AlternativeComparisonReport;
+using ViewModel.Utilities;
+
+namespace ViewModel.Saving.PersistenceManagers
+{
+    public class AlternativeComparisonReportPersistenceManager : SavingBase, IElementManager
+    {
+        private const int XML_COL = 2;
+        public override string TableName => "alternative_comparison_reports";
+
+        public override string[] TableColumnNames => new string[] { NAME, "xml" };
+
+        public override Type[] TableColumnTypes => new Type[] { typeof(string), typeof(string) };
+
+        public AlternativeComparisonReportPersistenceManager(Study.FDACache studyCache)
+        {
+            StudyCacheForSaving = studyCache;
+        }
+
+        public override ChildElement CreateElementFromRowData(object[] rowData)
+        {
+            string xml = (string)rowData[XML_COL];
+            AlternativeComparisonReportElement elem = new AlternativeComparisonReportElement(xml);
+            return elem;
+        }
+
+        #region Logging
+        //todo: not sure what to do with logging stuff yet. Waiting until a later task to determine
+        //if this should all get removed from the base class or not.
+        public ObservableCollection<LogItem> GetLogMessages(string elementName)
+        {
+            throw new NotImplementedException();
+        }
+
+        public ObservableCollection<LogItem> GetLogMessagesByLevel(LoggingLevel level, string elementName)
+        {
+            throw new NotImplementedException();
+        }
+
+        public void Log(LoggingLevel level, string message, string elementName)
+        {
+            throw new NotImplementedException();
+        }
+        #endregion
+
+        public override object[] GetRowDataFromElement(ChildElement elem)
+        {
+            object[] retval = new object[] { elem.Name, ((AlternativeComparisonReportElement)elem).WriteToXML() };
```

Comment:
Check the class or make WriteToXml an interface method on ChildElement

Reply:
done.